tox.ini

```diff
@@ -1,67 +1,68 @@
 [tox]
+# skip install to avoid rebuilding the world, do the install with setup.py
+# instead of pip below
+skip_install = True
 # these are the environments that will run when you
 # execute `tox` in the command-line
-# Note: linting is not included by default
 envlist =
     lint
+    build
     all-deps
     min-deps
     examples
-
-
-[testenv:all-deps]
-# skip install to avoid rebuilding the world, do the install with setup.py
-# instead of pip below
-skip_install = True
```

Comment:
How do build time look without this? This was an optimization before pip used in-tree build to be able to re-use cached object files in `target`. You can check this by comparing the time it takes to do `pip install .` after `rm -rf target/` (no cache) and after `rm -rf target && cargo build --release`. If the second one is much smaller, then we don't need this `skip_install` anymore.

Reply:
oh, just realized you moved the setting on top of the tox.ini ...